### PR TITLE
WIFI-2088: radsec: fix cert download from filestore

### PR DESCRIPTION
**Pull request by ArifAlam**
Create cert directory per configured radsec server.

Signed-off-by: Arif Alam <arif.alam@netexperience.com>

### CODE REVIEW

File: feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radius_proxy.c [hunk 1]
```diff
@@ -140,21 +140,31 @@ static bool radsec_download_cert(char *cert_name, char *dir_name, char *cert_url
...
+	sprintf(dir_path, "/tmp/radsec/certs/%s", dir);
 	sprintf(path, "/tmp/radsec/certs/%s/%s", dir, name);
 
+	if (stat(dir_path, &stat_buf) == -1)
+	{
+		char cmd[200];
+		sprintf(cmd, "mkdir -p %s", dir_path);
+		system(cmd);
+	}
+
```
Comment (RickSommerville) [inline feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radius_proxy.c:163]: From the captive portal code:
	struct stat st = {0};
	if (stat(path, &st) == -1)
		mkdir(path, 0755);

**Review by RickSommerville — COMMENTED** [review]
Looks good.  1 minor comment

File: feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radius_proxy.c (hunk 1 above)
Comment (ArifAlam) [reply feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radius_proxy.c:163]: mkdir() doesn't work for making nested directories in one shot.

**Comment by RickSommerville** [comment]
Merged 666e740e, thanks.

**Pull request closed without merging**